index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Code & Coffee</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato">
+    <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/cocogoose">
+    <link rel="icon" href="assets/icon.png">
+    <script src="https://kit.fontawesome.com/fb021b8662.js" crossorigin="anonymous"></script>  </head>
+  <body id="home">
+    <div id="menu">
+      <i class="fa-solid fa-x close"></i>
+        <ul>
+          <li class="navbar"><a href="about.html" onclick="toggleMenu()">About</a></li>
+          <li class="navbar"><a href="#program" onclick="toggleMenu()">Program</a></li>
+          <li class="navbar"><a href="#speakers" onclick="toggleMenu()">Speakers</a></li>
+          <li class="navbar"><a href="about.html#logo" onclick="toggleMenu()">Logo</a></li>
+          <li class="navbar"><a href="#sponsor" onclick="toggleMenu()">Sponsor</a></li>
+        </ul>
+    </div>
+    <header>
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <div>
+        <h6>English</h6>
+        <h6>My Page</h6>
+        <h6>Logout</h6>
+      </div>
+    </header>
+    <nav>
+      <div class="logo">
+        <img src="assets/intro/menu.png" class="burger" alt="Menu Button">
+      </div>
+      <div class="logo-desk">
+        <img src="assets/logo-color.png" alt="Code and Coffee Logo">
+      </div>
+      <div class="desktop-nav">
+        <ul class="nav-links">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#speakers">Speakers</a></li>
+          <li><a href="about.html#logo">Logo</a></li>
+          <li><a href="#sponsor">Sponsor</a></li>
+          <li><a href="#sponsor" id="campaign">C&C Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="intro">
+      <p>"Hello! Programmers"</p>
+      <h1>Code & Coffee <br>Cape Town</br> 2023</h1>
```

Comment:
Kindly make sure the font you use in your design look exactly the same as in the provided design 😄 , 
Even though the text should be customized it should look the same , it could be better if you improve that 👍 

| Your design | Cindy Design |
| :---: | :---| 
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/7c1de9e3-80bd-4b9e-a274-ba08cb527d2a) |  ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/96329faf-40dc-4a08-93b2-c93611364c6c) |


The same comment is applied to the `about` page , which I think also should be customized 👍 

| Your design | Cindy Design |
| :---: | :---| 
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/867e8225-5c01-455a-8ea8-6e4b5b59e572) | ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/4f7fc2dc-abb8-4f78-a369-5d86045303f8) |